tests/features/contact_form/submit_contact_form.feature

```diff
@@ -69,6 +68,7 @@ Feature: Submit the contact form
     And I should see the following success messages:
       | success messages                                              |
       | Your message has been submitted. Thank you for your feedback. |
+    And I am on the homepage
```

Comment:
Again, shouldn't this be `And I should be on the homepage`? What's the reason to navigate to homepage when in the next step you click on a link?